UnityProject/Assets/Scripts/Systems/Chemistry/Effects/ChemExplosion.cs

```diff
@@ -86,23 +86,33 @@ public override void Apply(MonoBehaviour sender, float amount)
 
 			if (strength > 0)
 			{
+				ExplosionNode node;
+				if (isEMP)
+				{
+					node = new ExplosionEmpNode();
```

Comment:
Wasn't the plan to use an enum?

Reply:
wait, what

Reply:
okay, I was dumb, chem explosion uses enum now